lib/vtls/schannel_verify.c

```diff
@@ -481,6 +518,12 @@ CURLcode Curl_verify_host(struct Curl_cfilter *cf,
   size_t hostlen = strlen(conn_hostname);
   DWORD len = 0;
   DWORD actual_len = 0;
+  PCERT_ALT_NAME_INFO alt_name_info = NULL;
+  DWORD alt_name_info_size = 0;
+  BYTE ip_blob[32] = { 0 };
+  BOOL Win8_compat;
+  PCRYPT_DATA_BLOB p = (PCRYPT_DATA_BLOB)ip_blob;
```

Comment:
Why create an array of bytes you point to like this, can't it be a `PCRYPT_DATA_BLOB` ?

If it needs to be an array, where is the '32' coming from?

Reply:
> Why create an array of bytes you point to like this, can't it be a PCRYPT_DATA_BLOB ?

We need a buffer that is 8 (IPv4) or 20 bytes (IPv6) in size. IPv4/6 addresses are 4/16 bytes respectively, and we need four additional bytes for cbData. The 32 byte buffer is more than enough to accommodate that.

Reply:
You should probably add that as a comment so that readers of the code can find this explanation.

If you need it to be 24 bytes, why do you make it 32?

Reply:
Maybe use `sizeof(PCRYPT_DATA_BLOB)` instead of the hardcoded number?